app/views/xur.html

```diff
@@ -4,10 +4,10 @@
     <div ng-repeat="category in vm.categoryOrder">
       <h2 ng-bind="category"></h2>
       <div class="items">
-        <div class="xurItem" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
+        <div class="xurItem" align="right" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
```

Comment:
We should use CSS - `align` is a real old HTML property that shouldn't be used anymore.